GoogleUtilities/AppDelegateSwizzler/README.md

```diff
@@ -132,6 +132,8 @@ MYAppDelegateInterceptor.m
 
 3. Register your interceptor when it makes sense to do so.
 
+// TODO: Update for [GULAppDelegateSwizzler proxyOriginalDelegateRemoteNotificationMethods]
+
```

Comment:
Fill in

Reply:
I added a separate section, just forgot to remove TODO.